core/src/library.rs

```diff
@@ -215,6 +225,12 @@ impl<'gc> MovieLibrary<'gc> {
         if let Some((id, character)) = self.character_by_export_name(export_name) {
             self.instantiate_display_object(id, character, mc)
         } else {
+            self.export_characters.iter().for_each(|(name, id)| {
+                tracing::error!("Exported character: {} -> {}", name, id);
```

Comment:
I'm assuming this is for debugging purposes, since it doesn't look like a normal error condition. You may want to remove these trace lines?